### ISSUE TITLE
Can not load beyond 1 epoch

### ISSUE THREAD

**Issue by realDSH** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#1610]
I don't know how to deal with it, can you tell me how to fix it? Thanks a lot!

C:\Users\Sonho\Downloads\RVC1006AMD_Intel>runtime\python.exe infer-web.py --pycmd runtime\python.exe --port 7897 --dml
2023-12-12 22:29:29 | INFO | configs.config | No supported Nvidia GPU found
overwrite preprocess and configs.json
2023-12-12 22:29:29 | INFO | configs.config | Use DirectML instead
is_half:False, device:privateuseone:0
2023-12-12 22:29:32 | INFO | __main__ | Use Language: zh_CN
Running on local URL:  http://0.0.0.0:7897
2023-12-12 22:30:30 | INFO | __main__ | "runtime\python.exe" infer/modules/train/preprocess.py "C:\Users\Sonho\dshvoice" 40000 6 "C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice" False 3.7
['infer/modules/train/preprocess.py', 'C:\\Users\\Sonho\\dshvoice', '40000', '6', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice', 'False', '3.7']
start preprocess
['infer/modules/train/preprocess.py', 'C:\\Users\\Sonho\\dshvoice', '40000', '6', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice', 'False', '3.7']
['C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel\\infer\\modules\\train\\preprocess.py', 'C:\\Users\\Sonho\\dshvoice', '40000', '6', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice', 'False', '3.7']
['C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel\\infer\\modules\\train\\preprocess.py', 'C:\\Users\\Sonho\\dshvoice', '40000', '6', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice', 'False', '3.7']
['C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel\\infer\\modules\\train\\preprocess.py', 'C:\\Users\\Sonho\\dshvoice', '40000', '6', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice', 'False', '3.7']
['C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel\\infer\\modules\\train\\preprocess.py', 'C:\\Users\\Sonho\\dshvoice', '40000', '6', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice', 'False', '3.7']
['C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel\\infer\\modules\\train\\preprocess.py', 'C:\\Users\\Sonho\\dshvoice', '40000', '6', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice', 'False', '3.7']
['C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel\\infer\\modules\\train\\preprocess.py', 'C:\\Users\\Sonho\\dshvoice', '40000', '6', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice', 'False', '3.7']
C:\Users\Sonho\dshvoice/dshpeace.wav->Suc.
end preprocess
2023-12-12 22:30:54 | INFO | __main__ | start preprocess
['infer/modules/train/preprocess.py', 'C:\\Users\\Sonho\\dshvoice', '40000', '6', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice', 'False', '3.7']
C:\Users\Sonho\dshvoice/dshpeace.wav->Suc.
end preprocess

2023-12-12 22:30:54 | INFO | __main__ | runtime\python.exe infer/modules/train/extract/extract_f0_rmvpe_dml.py "C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice"
['infer/modules/train/extract/extract_f0_rmvpe_dml.py', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice']
todo-f0-233
f0ing,now-0,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_0.wav
Loading rmvpe model
2023-12-12 22:30:58.6777356 [W:onnxruntime:, session_state.cc:1169 onnxruntime::VerifyEachNodeIsAssignedToAnEp] Some nodes were not assigned to the preferred execution providers which may or may not have an negative impact on performance. e.g. ORT explicitly assigns shape related ops to CPU to improve perf.
2023-12-12 22:30:58.6840853 [W:onnxruntime:, session_state.cc:1171 onnxruntime::VerifyEachNodeIsAssignedToAnEp] Rerunning with verbose output on a non-minimal build will show node assignments.
f0ing,now-46,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_150.wav
f0ing,now-92,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_203.wav
f0ing,now-138,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_254.wav
f0ing,now-184,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_35.wav
f0ing,now-230,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_97.wav
2023-12-12 22:33:28 | INFO | __main__ | ['infer/modules/train/extract/extract_f0_rmvpe_dml.py', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice']
todo-f0-233
f0ing,now-0,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_0.wav
f0ing,now-46,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_150.wav
f0ing,now-92,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_203.wav
f0ing,now-138,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_254.wav
f0ing,now-184,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_35.wav
f0ing,now-230,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_97.wav

2023-12-12 22:33:28 | INFO | __main__ | "runtime\python.exe" infer/modules/train/extract_feature_print.py privateuseone:0 1 0  "C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice" v2
['infer/modules/train/extract_feature_print.py', 'privateuseone:0', '1', '0', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice', 'v2']
C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice
load model(s) from assets/hubert/hubert_base.pt
move model to privateuseone:0
all-feature-233
now-233,all-0,0_0.wav,(184, 768)
now-233,all-23,0_125.wav,(127, 768)
now-233,all-46,0_150.wav,(89, 768)
now-233,all-69,0_178.wav,(32, 768)
now-233,all-92,0_203.wav,(40, 768)
now-233,all-115,0_229.wav,(194, 768)
now-233,all-138,0_254.wav,(184, 768)
now-233,all-161,0_282.wav,(132, 768)
now-233,all-184,0_35.wav,(184, 768)
now-233,all-207,0_70.wav,(60, 768)
now-233,all-230,0_97.wav,(60, 768)
all-feature-done
2023-12-12 22:38:24 | INFO | __main__ | ['infer/modules/train/extract/extract_f0_rmvpe_dml.py', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice']
todo-f0-233
f0ing,now-0,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_0.wav
f0ing,now-46,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_150.wav
f0ing,now-92,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_203.wav
f0ing,now-138,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_254.wav
f0ing,now-184,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_35.wav
f0ing,now-230,all-233,-C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice/1_16k_wavs/0_97.wav
['infer/modules/train/extract_feature_print.py', 'privateuseone:0', '1', '0', 'C:\\Users\\Sonho\\Downloads\\RVC1006AMD_Intel/logs/myvoice', 'v2']
C:\Users\Sonho\Downloads\RVC1006AMD_Intel/logs/myvoice
load model(s) from assets/hubert/hubert_base.pt
move model to privateuseone:0
all-feature-233
now-233,all-0,0_0.wav,(184, 768)
now-233,all-23,0_125.wav,(127, 768)
now-233,all-46,0_150.wav,(89, 768)
now-233,all-69,0_178.wav,(32, 768)
now-233,all-92,0_203.wav,(40, 768)
now-233,all-115,0_229.wav,(194, 768)
now-233,all-138,0_254.wav,(184, 768)
now-233,all-161,0_282.wav,(132, 768)
now-233,all-184,0_35.wav,(184, 768)
now-233,all-207,0_70.wav,(60, 768)
now-233,all-230,0_97.wav,(60, 768)
all-feature-done

2023-12-12 22:38:24 | INFO | __main__ | Use gpus:
2023-12-12 22:38:24 | INFO | __main__ | "runtime\python.exe" infer/modules/train/train.py -e "myvoice" -sr 40k -f0 1 -bs 1 -te 20 -se 5 -pg assets/pretrained_v2/f0G40k.pth -pd assets/pretrained_v2/f0D40k.pth -l 1 -c 0 -sw 1 -v v2
NO GPU DETECTED: falling back to CPU - this may take a while
INFO:myvoice:{'data': {'filter_length': 2048, 'hop_length': 400, 'max_wav_value': 32768.0, 'mel_fmax': None, 'mel_fmin': 0.0, 'n_mel_channels': 125, 'sampling_rate': 40000, 'win_length': 2048, 'training_files': './logs\\myvoice/filelist.txt'}, 'model': {'filter_channels': 768, 'gin_channels': 256, 'hidden_channels': 192, 'inter_channels': 192, 'kernel_size': 3, 'n_heads': 2, 'n_layers': 6, 'p_dropout': 0, 'resblock': '1', 'resblock_dilation_sizes': [[1, 3, 5], [1, 3, 5], [1, 3, 5]], 'resblock_kernel_sizes': [3, 7, 11], 'spk_embed_dim': 109, 'upsample_initial_channel': 512, 'upsample_kernel_sizes': [16, 16, 4, 4], 'upsample_rates': [10, 10, 2, 2], 'use_spectral_norm': False}, 'train': {'batch_size': 1, 'betas': [0.8, 0.99], 'c_kl': 1.0, 'c_mel': 45, 'epochs': 20000, 'eps': 1e-09, 'fp16_run': False, 'init_lr_ratio': 1, 'learning_rate': 0.0001, 'log_interval': 200, 'lr_decay': 0.999875, 'seed': 1234, 'segment_size': 12800, 'warmup_epochs': 0}, 'model_dir': './logs\\myvoice', 'experiment_dir': './logs\\myvoice', 'save_every_epoch': 5, 'name': 'myvoice', 'total_epoch': 20, 'pretrainG': 'assets/pretrained_v2/f0G40k.pth', 'pretrainD': 'assets/pretrained_v2/f0D40k.pth', 'version': 'v2', 'gpus': '0', 'sample_rate': '40k', 'if_f0': 1, 'if_latest': 1, 'save_every_weights': '1', 'if_cache_data_in_gpu': 0}
INFO:torch.distributed.distributed_c10d:Added key: store_based_barrier_key:1 to store for rank: 0
INFO:torch.distributed.distributed_c10d:Rank 0: Completed store-based barrier for key:store_based_barrier_key:1 with 1 nodes.
DEBUG:infer.lib.infer_pack.models:gin_channels: 256, self.spk_embed_dim: 109
INFO:myvoice:loaded pretrained assets/pretrained_v2/f0G40k.pth
INFO:myvoice:<All keys matched successfully>
INFO:myvoice:loaded pretrained assets/pretrained_v2/f0D40k.pth
INFO:myvoice:<All keys matched successfully>
C:\Users\Sonho\Downloads\RVC1006AMD_Intel\runtime\lib\site-packages\torch\functional.py:641: UserWarning: stft with return_complex=False is deprecated. In a future pytorch release, stft will return complex tensors for all inputs, and return_complex=False will raise an error.
Note: you can still call torch.view_as_real on the complex output to recover the old return format. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\SpectralOps.cpp:867.)
  return _VF.stft(input, n_fft, hop_length, win_length, window,  # type: ignore[attr-defined]
C:\Users\Sonho\Downloads\RVC1006AMD_Intel\runtime\lib\site-packages\torch\functional.py:641: UserWarning: stft with return_complex=False is deprecated. In a future pytorch release, stft will return complex tensors for all inputs, and return_complex=False will raise an error.
Note: you can still call torch.view_as_real on the complex output to recover the old return format. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\SpectralOps.cpp:867.)
  return _VF.stft(input, n_fft, hop_length, win_length, window,  # type: ignore[attr-defined]
C:\Users\Sonho\Downloads\RVC1006AMD_Intel\runtime\lib\site-packages\torch\functional.py:641: UserWarning: stft with return_complex=False is deprecated. In a future pytorch release, stft will return complex tensors for all inputs, and return_complex=False will raise an error.
Note: you can still call torch.view_as_real on the complex output to recover the old return format. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\SpectralOps.cpp:867.)
  return _VF.stft(input, n_fft, hop_length, win_length, window,  # type: ignore[attr-defined]
C:\Users\Sonho\Downloads\RVC1006AMD_Intel\runtime\lib\site-packages\torch\functional.py:641: UserWarning: stft with return_complex=False is deprecated. In a future pytorch release, stft will return complex tensors for all inputs, and return_complex=False will raise an error.
Note: you can still call torch.view_as_real on the complex output to recover the old return format. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\SpectralOps.cpp:867.)
  return _VF.stft(input, n_fft, hop_length, win_length, window,  # type: ignore[attr-defined]
C:\Users\Sonho\Downloads\RVC1006AMD_Intel\runtime\lib\site-packages\torch\functional.py:641: UserWarning: stft with return_complex=False is deprecated. In a future pytorch release, stft will return complex tensors for all inputs, and return_complex=False will raise an error.
Note: you can still call torch.view_as_real on the complex output to recover the old return format. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\SpectralOps.cpp:867.)
  return _VF.stft(input, n_fft, hop_length, win_length, window,  # type: ignore[attr-defined]
INFO:torch.nn.parallel.distributed:Reducer buckets have been rebuilt in this iteration.
C:\Users\Sonho\Downloads\RVC1006AMD_Intel\runtime\lib\site-packages\torch\autograd\__init__.py:200: UserWarning: Grad strides do not match bucket view strides. This may indicate grad was not created according to the gradient layout contract, or that the param's strides changed since DDP was constructed.  This is not an error, but may impair performance.
grad.sizes() = [1, 21, 96], strides() = [70560, 96, 1]
bucket_view.sizes() = [1, 21, 96], strides() = [2016, 96, 1] (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\torch\csrc\distributed\c10d\reducer.cpp:337.)
  Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
INFO:myvoice:Train Epoch: 1 [0%]
INFO:myvoice:[0, 0.0001]
INFO:myvoice:loss_disc=3.737, loss_gen=4.677, loss_fm=12.644,loss_mel=27.074, loss_kl=8.659
DEBUG:matplotlib:matplotlib data path: C:\Users\Sonho\Downloads\RVC1006AMD_Intel\runtime\lib\site-packages\matplotlib\mpl-data
DEBUG:matplotlib:CONFIGDIR=C:\Users\Sonho\.matplotlib
DEBUG:matplotlib:interactive is False
DEBUG:matplotlib:platform is win32
INFO:torch.nn.parallel.distributed:Reducer buckets have been rebuilt in this iteration.
